CONTRIBUTING.md

```diff
@@ -0,0 +1,86 @@
+# Contributing to I-Ching-app
+
+First off, thank you for considering contributing to the I-Ching-app! 🎉 We value your time and effort. Whether you're fixing a bug, proposing a new feature, improving documentation, or writing code, your contribution is welcome and appreciated.
+
+This document provides guidelines for contributing to the I-Ching-app. Please take a moment to review it to ensure a smooth and effective contribution process.
+
+## Code of conduct
```

Comment:
IDK, I'm kinda' neutral on CoC ... but .. I guess it can make sense ... I was actually thinking of moving this under the encorpora "monorepo" - https://github.com/corpora-inc/encorpora ... but, each app having its own issues could be a good thing ...

I'm generally low-maintenance, low-effort on things like this .. I think it can be fine though ...

We probably do need a license though ... I guess I wort of lean APGL like I did with the main `corpora` software. The idea there is that teams can use it as they like internally but then they may love it so much that they want to get commercial support for their deployment at scale ... 🤔 

Anyways, I think we can merge this but I'm kinda' an oddball politically - mixed feelings on CoCs in general :D